singular.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * The template for displaying all pages, single posts and attachments
+ *
+ * This is a new template file that WordPress introduced in
+ * version 4.2. Note that it uses conditional logic to display
```

Comment:
While I agree wholeheartedly regarding not facilitating use of older core
versions, having a file named "singular.php" will not cause any issues with
pre-4.3 versions of WordPress. The file will simply be ignored by the
template hierarchy. Both page.php and single.php will fall back to
index.php.

On Wed, Sep 16, 2015 at 4:04 PM, Takashi Irie <notifications@github.com>
wrote:

> In singular.php
> <https://github.com/WordPress/twentysixteen/pull/165#discussion_r39677871>
> :
>
> > @@ -0,0 +1,64 @@
> > +<?php
> > +/**
> > + * The template for displaying all pages, single posts and attachments
> > + *
> > + * This is a new template file that WordPress introduced in
> > + * version 4.2. Note that it uses conditional logic to display
>
> This template was introduced in 4.3, wasn't it? Although it won't give us
> a fatal error, we probably should stop the theme to be activated below 4.3
> to make sure the theme work as it supposed to be.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WordPress/twentysixteen/pull/165/files#r39677871>.
>